app/src/views/private/components/license-banner.vue

```diff
@@ -0,0 +1,166 @@
+<script setup lang="ts">
+import { useI18n } from 'vue-i18n';
+import { useSettingsStore } from '@/stores/settings';
+import { useServerStore } from '@/stores/server';
+import { storeToRefs } from 'pinia';
+import BannerSVG from '../../../assets/directus-bsl-banner.svg?raw';
+
+const { t } = useI18n();
+
+const settingsStore = useSettingsStore();
+
+async function acceptTerms() {
+	await settingsStore.updateSettings({ license_banner_seen: true });
+	settingsStore.hydrate();
+}
+
+const { info } = storeToRefs(useServerStore());
+</script>
+
+<template>
+	<v-dialog>
+		<v-card>
+			<div class="inner">
+				<!-- eslint-disable-next-line vue/no-v-html -->
+				<div class="banner-svg" v-html="BannerSVG"></div>
```

Comment:
To be inline with other parts of the codebase[^1][^2][^3] should we use an `img` tag to render the svg? This would remove the need for `v-html`


[^1]: https://github.com/directus/directus/blob/main/app/src/views/public/public-view.vue#L65
[^2]: https://github.com/directus/directus/blob/main/app/src/displays/user/user.vue#L39
[^3]: https://github.com/directus/directus/blob/bc60657fc33587a0961164fc3d7ddffcd274c789/app/src/views/shared/shared-view.vue#L43

Reply:
We're using dynamic color vars in the svg, which requires it to be imported with `?raw`, which to my knowledge doesn't work with `img` tag 🤔, but I'll take another look to confirm.